Action.podspec

```diff
@@ -20,8 +20,8 @@ Pod::Spec.new do |s|
   s.source_files  = "Sources/**/*.{swift}"
 
   s.frameworks  = "Foundation"
-  s.dependency "RxSwift", "~> 4.0.0-rc.0"
-  s.dependency "RxCocoa", "~> 4.0.0-rc.0"
+  s.dependency "RxSwift", "~> 4.0.0"
+  s.dependency "RxCocoa", "~> 4.0.0"
```

Comment:
Regardless of the version we decide to go with for Action, these should really be `~> 4.0` instead of `~> 4.0.0`. Otherwise, we will need to update the podspec again when RxSwift 4.1 comes out. Let me know if I can clarify anything!